lib/order.rb

```diff
@@ -0,0 +1,7 @@
+class Order
+def add(dish, quantity)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `dish`. If it's necessary, use `_` or `_dish` as an argument name to indicate that it won't be used. You can also write as `add(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)<br>Lint/UnusedMethodArgument: Unused method argument - `quantity`. If it's necessary, use `_` or `_quantity` as an argument name to indicate that it won't be used. You can also write as `add(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)